src/binary/ibc-setup/commands/ics20.spec.ts

```diff
@@ -0,0 +1,138 @@
+import fs from 'fs';
+import path from 'path';
+
+import { assert } from '@cosmjs/utils';
+import test from 'ava';
+import sinon from 'sinon';
+
+import { appFile } from '../../constants';
+
+import { createClient, Options, run } from './ics20';
+
+const fsWriteFileSync = sinon.stub(fs, 'writeFileSync');
+const fsReadFileSync = sinon.stub(fs, 'readFileSync');
+const consoleLog = sinon.stub(console, 'log');
+
+const mnemonic =
+  'enlist hip relief stomach skate base shallow young switch frequent cry park';
+
+const registryYaml = `
+version: 1
+
+chains:
+  musselnet:
```

Comment:
You can remove musselnet from all these tests, we will only use the two local (ci) chains

Reply:
4dfc837